R/utils_accessibility.R

```diff
@@ -121,6 +121,6 @@ h6_tabstop <- function(header, tabindex = 0, ...) {
 nhs_card_tabstop <- function(header, tabindex = 0, ...) {
   # create nhs_card as typical nhs_card plus tabindex attribute
   # ensures nhs_card will be stopped at when pressing keyboard tab
-  nhs_card_tabstop <- div(nhs_card(header, ...)) %>%
+  nhs_card_tabstop <- nhs_card(header, ...) %>%
```

Comment:
I had previously added this div to wrap everything and cannot remember why. Anyway, it was preventing the chart from displaying so I have removed.

Reply:
without div it shows '0'!  I remember when you showed me scrollytell example, there were strange 0 appeared and somehow div solved that issue.??

Reply:
However it only shows 0 if I put two charts inside nhs_col_2 etc..

Reply:
I'll have a look into what is going on. May need to compare what we see, as I suspect the behaviour is different for you. Package versions perhaps?

Reply:
This should be sorted in latest commit.